contracts/interfaces/rysk/IRyskLiquidityPool.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.10;
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+/**
+ * Describes the interface used to communicate with the Rysk LiquidityPool contract.
+ *
+ */
+interface IRyskLiquidityPool is IERC20 {
+    /**
+     * Gets the current epoch.
+     *
+     * @notice Epochs represent time resolution on which the pool operates.
+     * For every epoch a new price per share is calculated and pool rebalances.
+     * Withdrawals are initiated in one epoch and completed in some later epoch.
+     *
+     * @return the current epoch number
+     */
+    function epoch() external view returns (uint256);
```

Comment:
we've changed how the epoch system works so theres now a depositEpoch and a withdrawEpoch, this is so we can maintain solvency and manage funds completely. For the changes refer to: https://github.com/rysk-finance/dynamic-hedging/blob/rysk-alpha/packages/contracts/contracts/LiquidityPool.sol#L59

Reply:
thanks for pointing this out, I've made modifications to the strategy to accommodate this